redbot.py

```
@@ -220,8 +262,89 @@ def find_subdomains(self, domain: str, wordlist: List[str] = None) -> List[str]:
+
+                # Configura user agent
+                page.set_extra_http_headers({
+                    'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36'
+                })
+
+                page.goto(url, timeout=30000)
+
+                # Coleta informações
+                title = page.title()
+                url_final = page.url
+                page_source = page.content()
```

Ruff (0.13.1)

Local variable `page_source` is assigned to but never used

Remove assignment to unused variable `page_source`

(F841)